slcp/process.py

```diff
@@ -54,12 +59,15 @@ def select_source(self):
         if not ask user for input using filedialog.
         :return: str. Source folder path.
         """
+
+        root = tk.Tk()
+        root.withdraw()
```

Comment:
I think these lines should be under `if self.args.source is None` condition below. This way we will not create `Tk` instance when it is not needed.

Reply:
You're right again.